coalib/coala_delete_orig.py

```diff
@@ -1,21 +1,17 @@
+import logging
 import os
 
-from pyprint.ConsolePrinter import ConsolePrinter
-
 from coalib.output.Logging import configure_logging
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.parsing import Globbing
+from coalib.parsing.Globbing import glob_escape
 from coalib.settings.ConfigurationGathering import get_config_directory
 from coalib.settings.Section import Section
-from coalib.parsing.Globbing import glob_escape
 
 
-def main(log_printer=None, section: Section=None):
+def main(section: Section=None):
```

Comment:
`git grep "delete_orig"` returned this.
```
coalib/coala_delete_orig.py:    return start_delete_orig(section)
coalib/coala_delete_orig.py:def start_delete_orig(section: Section=None):
setup.py:                  'coala-delete-orig = coalib.coala_delete_orig:main']},
tests/coalaDeleteOrigTest.py:from coalib import coala_delete_orig
tests/coalaDeleteOrigTest.py:        retval = coala_delete_orig.start_delete_orig()
tests/coalaDeleteOrigTest.py:            retval = coala_delete_orig.main(section=self.section)
tests/coalaDeleteOrigTest.py:            retval = coala_delete_orig.main(section=self.section)
tests/coalaDeleteOrigTest.py:            retval = coala_delete_orig.main(section=section)
```

I am not completly sure what you mean by signature, by no, we dont call this function from anywhere else in coala.